src/js/modules/http-services/themoviedb-api.js

```diff
@@ -8,11 +9,12 @@ export class ThemoviedbApi {
     this.page = 1;
     axios.defaults.baseURL = 'https://api.themoviedb.org/';
     this.trends = 'popular';
+    this.lang = `${this.getCurrentClientLang()}`;
   }
   async getMovies() {
     try {
       const response = await axios.get(
-        `/3/movie/${this.trends}?api_key=${this.API_KEY}&language=en-US&page=${this.page}&region=UA`,
+        `/3/trending/movie/day?api_key=${this.API_KEY}&language=${this.lang}&page=${this.page}`,
```

Comment:
Why did you change this part?
It may be necessary for filtering functionality - please consult with Bogdan and reply to this comment?